t/plugin/openid-connect.t

```diff
@@ -1241,7 +1345,7 @@ passed
 
 
 
-=== TEST 33: Access route to validate "x-userinfo" in request header
+=== TEST 35: Access route to validate "x-userinfo" in request header
```

Comment:
I suggest put your newly added test cases to the end of the file, so that you won't need to modify the index of the rest test cases. 
The index of test cases below this one are not correct

Reply:
Oh of course, I'm not sure how I missed that. I've updated the test file.